### DISCUSSION TITLE
Changelog behavior for stable/pre-releases

### DISCUSSION THREAD

**Discussion by seriousm4x** [goreleaser/goreleaser#4336] [Q&A] [answered]
Hi, is there a possibility that the changelog for stable releases is always created since the last stable release?

The following scenario:

1. `Stable release` - changelog is created
2. Multiple commits
3. `Pre-release` - changelog is created since step 1
4. Multiple commits
5. `Stable release`  - changelog is created since step 3

Now the last stable release contains only the changelog since the last pre-release (step 3), but I would like to have the whole changelog since the last stable release (step 1). Is that possible?

**Comment by caarlos0** [comment, marked as answer]
https://goreleaser.com/customization/git/

ignore tags might work I think

**Reply by seriousm4x** [reply]
Something like this?
```yml
git:
  ignore_tags:
    - "{{ if not .prerelease}}beta{{ end }}"
```
Found [here](https://github.com/goreleaser/goreleaser/blob/main/.goreleaser.yaml#L20-L22)

**Reply by caarlos0** [reply]
yup!
